concent_api/utils/logging.py

```diff
@@ -200,8 +203,9 @@ def log_receive_message_from_database(message: Message, client_public_key: bytes
     ))
 
 
-def log_file_status(subtask_id: str, requestor_public_key: bytes, provider_public_key: bytes):
-    logger.info('File assigned to subtask with {} id already uploaded. -- REQUESTOR PUBLIC KEY: {} -- PROVIDER PUBLIC KEY {}'.format(
+def log_file_status(task_id: str, subtask_id: str, requestor_public_key: bytes, provider_public_key: bytes):
+    logger.info('File assigned to TASK_ID: {} SUBTASK_ID: {} is already uploaded. -- REQUESTOR PUBLIC KEY: {} -- PROVIDER PUBLIC KEY {}'.format(
+        task_id,
```

Comment:
Previous pull request was accidentally closed. Radek, you asked about  log_file_status, it should be changed as well and  task_id should be add, i did it.